aissemble-open-inference-protocol-kserve/tests/features/steps/converter_steps.py

```diff
@@ -0,0 +1,119 @@
+from behave import given, then, when
+
+from kserve import InferRequest, InferInput, InferResponse, InferOutput
+from aissemble_open_inference_protocol_shared.types.dataplane import (
+    Parameters,
+    InferenceRequest,
+    InferenceResponse,
+    RequestInput,
+    RequestOutput,
+    ResponseOutput,
+    Datatype,
+)
+from kserve.protocol.infer_type import RequestedOutput
+
+from aissemble_open_inference_protocol_kserve.converters.converter import (
+    convert_infer_request,
+    convert_inference_response,
+)
+
+
+@given("an InferRequest with KServe")
+def given_infer_request_with_kserve(context):
+    infer_input = InferInput(
+        name="input",
+        shape=[1],
+        datatype="FP32",
+        data=[1, 2],
+        parameters=Parameters(),
+    )
+    infer_output = RequestedOutput(
+        name="output",
+        parameters=Parameters(),
+    )
+    context.infer_request = InferRequest(
+        model_name="test",
+        request_id="1",
+        infer_inputs=[infer_input],
+        from_grpc=False,
+        parameters={},
+        request_outputs=[infer_output],
+    )
+
+
+@given("an InferenceResponse with FASTAPI")
+def given_inference_response_with_fastapi(context):
+    infer_output = ResponseOutput(
+        name="output",
+        shape=[1],
+        datatype=Datatype.FP32,
+        data=[11, 22],
+        parameters=Parameters(),
+    )
+    context.inference_response = InferenceResponse(
+        model_name="output",
+        modeL_version="v1",
```

Comment:
```suggestion
        model_version="v1",
```
Capital L to lowercase